llmfoundry/data/text_data.py

```diff
@@ -23,7 +23,7 @@
 from transformers import PreTrainedTokenizerBase
 
 from llmfoundry import registry
-from llmfoundry.data import (
+from llmfoundry.data.data import (
```

Comment:
Can we add these exports to the parent class `__init__` so that we don't need to import from `data.data`?